docs/user/alerting/rule-types.asciidoc

```diff
@@ -15,45 +15,64 @@ see {subscriptions}[the subscription page].
 [[stack-rules]]
 === Stack rules
 
-<<create-and-manage-rules, Stack rules>> are built into {kib}.  To access the *Stack Rules* feature and create and edit rules, users require the `all` privilege. See <<kibana-feature-privileges, feature privileges>> for more information.
+<<create-and-manage-rules, Stack rules>> are built into {kib}. To access the *Stack Rules* feature and create and edit rules, users require the `all` privilege. See <<kibana-feature-privileges, feature privileges>> for more information. In the full rules list in the UI, Stack rules are shown under the proper category `STACK RULES`.
 
 [cols="2*<"]
 |===
 
-| <<rule-type-index-threshold>>
-| Aggregate field values from documents using {es} queries, compare them to threshold values, and schedule actions to run when the thresholds are met.
-
 | <<rule-type-es-query>>
 | Run a user-configured {es} query, compare the number of matches to a configured threshold, and schedule actions to run when the threshold condition is met.
 
+| <<rule-type-index-threshold>>
+| Aggregate field values from documents using {es} queries, compare them to threshold values, and schedule actions to run when the thresholds are met.
+
 | {ref}/transform-alerts.html[{transform-cap} rules] beta:[]
 | beta:[] Run scheduled checks on a {ctransform} to check its health. If a {ctransform} meets the conditions, an alert is created and the associated action is triggered.
 
+| <<geo-alerting, Tracking containment>>
+| Run an {es} query to determine if any documents are currently contained in any boundaries from a specified boundary index and generate alerts when a rule's conditions are met.
+
 |===
 
 [float]
-[[domain-specific-rules]]
-=== Domain rules
+[[observability-rules]]
+=== Observability rules
 
-Domain rules are registered by *Observability*, *Security*, <<maps, Maps>> and <<xpack-ml, Machine Learning>>.
+In the rules list in the Management UI, Observability rules are shown under the multiple categories `APM AND USER EXPERIENCE`, `LOGS`, `METRICS`, `STACK MONITORING` and `UPTIME`.
```

Comment:
Maybe something a little more generic:

Observability rules categorized into APM and User Expereince, Logs, Metrics, Stack Monitoring, and Uptime.